etc/rollup/config.js

```diff
@@ -6,7 +6,7 @@ import url from 'rollup-plugin-url';
 import babel from 'rollup-plugin-babel';
 import commonjs from 'rollup-plugin-commonjs';
 import { terser } from 'rollup-plugin-terser';
-import json from '@rollup/plugin-json';
+import css from 'rollup-plugin-css-only';
```

Comment:
Have you tested the compile command? I think it will not work since you still have `json()` in line 34.

Reply:
ugh no 🤦🏻‍♂️ , on it 👍

Reply:
Hmm seems fine after removing it, however, I see this warning 🧐
```
$ rollup --config ./etc/rollup/config.js

/Users/lluis.agusti/Code/zopa-react-components/src/index.ts → cjs, es...
(!) Mixing named and default exports
https://rollupjs.org/guide/en/#output-exports
The following entry modules are using named and default exports together:
src/components/atoms/Dropdown/Dropdown.tsx
node_modules/babel-preset-react-app/node_modules/@babel/runtime/regenerator/index.js

Consumers of your bundle will have to use chunk['default'] to access their default export, which may not be what you want. Use `output.
exports: 'named'` to disable this warning
[!] (plugin css) Error: ENOENT: no such file or directory, mkdir
Error: ENOENT: no such file or directory, mkdir
```

Reply:
This happens also in master so it can be addressed in another PR